libyul/backends/evm/EVMDialect.cpp

```diff
@@ -111,11 +111,34 @@ pair<YulString, BuiltinFunctionForEVM> createFunction(
 	return {name, f};
 }
 
+namespace {
+// TODO: merge this completely into createBuiltins?
```

Comment:
Should I merge this into `createBuiltins`?

Reply:
I'd say yes

Reply:
Done.